### PR TITLE
changed the overflow property of the app.style.js file to hidden

### LINKED ISSUE #5721: Website not correctly aligned according to different devices.
### Description
I use a 6.2 inch phone and it does not correctly align to the page and it keeps zooming in everytime. 

### Expected Behavior
The website's should be more responsive and more aligned to the dimensions of the devices. 

### Environment:
- Host OS: XOS V13.0.0
- Browser: Chrome

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by BHAV0207**
**Description**

This PR fixes #5721 

**Notes for Reviewers**
In this PR, I explicitly added styling to the html tag which fixes the responsiveness issue for Devices of different screen sizes. The website's has become more responsive and more aligned to the dimensions of the devices. 

- [x]   Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ceca5eda3ca3c1fdf6d1efbc27a25a51d94b0e9e at: https://66a3b7e224609a0075d3fa00--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 16c43016feed93432e6bd03e5af03dc428a32130 at: https://66a3c8489bd4590eed255d15--layer5.netlify.app

**Comment by Ashparshp** [comment]
@BHAV0207 Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

File: src/sections/app.style.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ const GlobalStyle = createGlobalStyle`
   }
 
 html{
+    overflow-x: hidden;
```
Comment (sudhanshutech) [inline src/sections/app.style.js:46]: have you tested this all pages and its not having negative effect right?

Comment (BHAV0207) [reply src/sections/app.style.js:46]: yes I have tested all pages , on my side there are no negative effects

**Review by Ashparshp — APPROVED** [review]
LGTM!! Thanks for making the necessary changes, as this solves the overflow issue for all pages. Great work, @BHAV0207!!

**Comment by amitamrutiya** [comment]
Hey @sudhanshutech I found one side effect :
https://66a3c8489bd4590eed255d15--layer5.netlify.app/cloud-native-management/meshery/operating-cloud-native-infra

On this page, the right part of the person's image is hidden due to these changes.

image

Wait for @Ashparshp confirmation before making any changes.

**Comment by Ashparshp** [comment]
@amitamrutiya Thanks, but can you please confirm the width for which you are facing this issue?

**Comment by Ashparshp** [comment]
> @amitamrutiya Thanks, but can you please confirm the width for which you are facing this issue?

Also, I will ask you to recheck as everything seems good on my side.

**Comment by amitamrutiya** [comment]
Any standard mobile screen size around 450px.

**Comment by amitamrutiya** [comment]
Weird, I am facing this issue in Firefox but not in Chrome.🤔

**Comment by Ashparshp** [comment]
Thank you @amitamrutiya. But everything looks good on my side. Please recheck from your side and confirm.

**Comment by Ashparshp** [comment]
Could you please go through the page at [layer5.io//cloud-native-management/meshery/operating-cloud-native-infra](https://layer5.io/cloud-native-management/meshery/operating-cloud-native-infra) and check if the issue still exists? It might not be related to this PR.

**Comment by amitamrutiya** [comment]
I am checking on this link https://66a3c8489bd4590eed255d15--layer5.netlify.app/cloud-native-management/meshery/operating-cloud-native-infra at firefox. 

image

In chrome everything looks perfect.
image

**Comment by amitamrutiya** [comment]
By the way not a big issue. I think there is something not going well on my side. No worries :) ☺️

**Comment by Ashparshp** [comment]
The issue you are facing is not related to this PR; (it existed before this PR and persists in Firefox). Thank you so much for your review. Keep it up!

**Comment by Ashparshp** [comment]
@BHAV0207 Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by Muhammed-Moinuddin** [comment]
Specifically for the Homepage, what I found out is that just by removing one of the section i.e. (Deploy Faster Together With Confidence). The horizontal scrolling issue is resolved. So I think we just need to solve that section instead of adding it directly for the whole website.
@Ashparshp @sudhanshutech

**Comment by Ashparshp** [comment]
@Muhammed-Moinuddin, we're doing this because many pages in the handbook section have horizontal scrolling issues. By addressing this, we can fix all of them at once instead of solving for each page separately. I hope that's clear!

**Comment by BHAV0207** [comment]
@sudhanshutech  can you please review the changes

**Comment by Ashparshp** [comment]
@Muhammed-Moinuddin, @hargunkaur286, @amitamrutiya, @BHAV0207, do you have anything to say, or are we good to go with this PR?

**Comment by BHAV0207** [comment]
> @Muhammed-Moinuddin, @hargunkaur286, @amitamrutiya, @BHAV0207, do you have anything to say, or are we good to go with this PR?

from my side it's good to go.

**Comment by Muhammed-Moinuddin** [comment]
> @Muhammed-Moinuddin, @hargunkaur286, @amitamrutiya, @BHAV0207, do you have anything to say, or are we good to go with this PR?

This seems fine to me. We can move forward with it.

But... I got a small question that if we look another side of the picture, how we are going to find out responsiveness issues created due to any components overflow? This will make us to ignore those issues even they existed due to not making them properly responsive etc...
Thoughts?

**Comment by amitamrutiya** [comment]
LGTM as well.

> But... I got a small question that if we look another side of the picture, how we are going to find out responsiveness issues created due to any components overflow? This will make us to ignore those issues even they existed due to not making them properly responsive etc...
Thoughts?

Unfortunately, we’re stuck with just manual testing for now. 😔🔧

**Comment by amitamrutiya** [comment]
I am not familiar with automation testing for this.

**Comment by Ashparshp** [comment]
@amitamrutiya, alright! No issues.

**Pull request merged**